config/routes.rb

```diff
@@ -1,22 +1,14 @@
 Webapp::Application.routes.draw do
 
-  resources :home do
+  resources :assignments, only: [] do
     collection do
-      get :welcome
-    end
-  end
-
-  resources :assignments do
-    collection do
-      get :index
```

Comment:
You are correct, it doesn't work but /volunteers/super_admin is where  it is linked from. I'll ask and then we need to update that link to where it is supposed to go.